### PR TITLE
Add Toggle for Minimap and Tooltips

### PR DESCRIPTION
**Pull request by mickr777**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Have you discussed this change with the InvokeAI team?
- [ ] Yes
- [x] No, because:
If its not useful, they do not have to use it 😄 
      
## Description
While I was still in the viewportcontrols.tsx
added Option to toggle off the minimap with default being on(true)
added Tooltips to the buttons in viewportcontrols.tsx

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

**Comment by mickr777** [comment]
image

File: invokeai/frontend/web/dist/locales/en.json [hunk 1]
```diff
@@ -688,6 +688,15 @@
...
+        "clearNodes": "Clear Nodes",
+        "zoomInNodes": "Zoom In",
+        "zoomoutNodes": "Zoom Out",
+        "fitViewportNodes": "Fit View",
+        "hideGraphNodes": "Hide Graph Overlay",
+        "showGraphNodes": "Show Graph Overlay",
+        "hideLegendNodes": "Hide Field Type Legend",
+        "showLegendNodes": "Show Field Type Legend",
+        "hideMinimapnodes": "Hide MiniMap",
+        "showMinimapnodes": "Show MiniMap"
```
Comment (blessedcoolant) [inline invokeai/frontend/web/dist/locales/en.json:700]: this should be in public/locales .. Not dist.

**Review by blessedcoolant — CHANGES_REQUESTED** [review]

File: invokeai/frontend/web/dist/locales/en.json (hunk 1 above)
Comment (mickr777) [reply invokeai/frontend/web/dist/locales/en.json:700]: opps, yes you are right

Comment (mickr777) [reply invokeai/frontend/web/dist/locales/en.json]: @blessedcoolant in right file now, thanks

**Review by blessedcoolant — APPROVED** [review]

**Pull request merged**